src/component/page/deltakerliste-veileder/DeltakerlisteVeilederPage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { useTabTitle } from '../../../hooks/use-tab-title'
+import { fetchDeltakerlisteVeileder } from '../../../api/tiltak-api'
+import { isNotStartedOrPending, isRejected, usePromise } from '../../../utils/use-promise'
+import { AxiosResponse } from 'axios'
+import { VeiledersDeltaker } from '../../../api/data/deltaker'
+import { SpinnerPage } from '../../felles/spinner-page/SpinnerPage'
+import { AlertPage } from '../../felles/alert-page/AlertPage'
+import styles from './DeltakerlisteVeilederPage.module.scss'
+import { DeltakerlisteVeilederTabell } from './DeltakerlisteVeilederTabell'
+
+export const DeltakerlisteVeilederPage = (): React.ReactElement => {
+	useTabTitle('Deltakerliste')
+
+	const fetchDeltakerlisteVeilederPromise = usePromise<AxiosResponse<VeiledersDeltaker[]>>(
+		() => fetchDeltakerlisteVeileder()
+	)
+
+	if (isNotStartedOrPending(fetchDeltakerlisteVeilederPromise)) {
+		return <SpinnerPage />
+	}
+
+	if (isRejected(fetchDeltakerlisteVeilederPromise)) {
+		return <AlertPage variant="error" tekst="Noe gikk galt" />
+	}
+
+	const deltakerlisteVeileder = fetchDeltakerlisteVeilederPromise.result.data
```

Comment:
Bør være en useEffect som lagrer data i useState tror jeg, litt usikker på om det faktisk gir noe , men slik jeg har lært det bør man gjøre det

Reply:
Er du sikker på det..? Jeg kan veldig lite om dette her, men jeg fant denne: https://beta.reactjs.org/learn/you-might-not-need-an-effect og her står det blant annet: "If there is no external system involved (for example, if you want to update a component’s state when some props or state change), you shouldn’t need an Effect.". 

For meg virker det som at useEffect skal brukes for sideeffekter som skal f.eks. oppdatere DOM eller gjøre andre ting som skjer etter at komponenten rendres :/

Reply:
Det funker fint uten (testet lokalt og i dev)